website/docs/docs/use-dbt-semantic-layer/setup-sl.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Set up the dbt Semantic Layer"
+id: setup-sl
+description: "Seamlessly set up the dbt Semantic Layer in dbt Cloud using intuitive nagivation."
+sidebar_label: "Set up your Semantic Layer"
+tags: [Semantic Layer]
+---
+
+<VersionBlock firstVersion="1.6">
+
+import NewSLChanges from '/snippets/_new-sl-changes.md';
+
+<NewSLChanges />
+
+With the dbt Semantic Layer, you'll be able to centrally define business metrics, reduce code duplication and inconsistency, create self-service in downstream tools, and more. Configure the dbt Semantic Layer in dbt Cloud to connect with your integrated partner tool. 
+
+## Prerequisites
+
+Before you set up the dbt Semantic Layer, make sure you meet the following:
```

Comment:
sugg: since "prereqs" imply "you need/you must have", you can remove this intro sentence for conciseness

```suggestion
```